src/modules/peek/Peek.FilePreviewer/Controls/DriveControl.xaml

```diff
@@ -68,29 +62,25 @@
         <Grid
             Grid.Row="1"
             Grid.ColumnSpan="2"
-            RowSpacing="5">
+            RowSpacing="4">
             <Grid.RowDefinitions>
                 <RowDefinition Height="auto" />
                 <RowDefinition Height="auto" />
             </Grid.RowDefinitions>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="auto" />
-                <ColumnDefinition Width="*" />
-            </Grid.ColumnDefinitions>
             <Border
                 x:Name="CapacityBar"
                 Grid.Row="0"
                 Grid.ColumnSpan="2"
-                Height="20"
+                Height="24"
                 Background="{ThemeResource AccentFillColorDisabledBrush}"
-                CornerRadius="10" />
+                CornerRadius="12" />
 
             <Border
                 Grid.Row="0"
                 Grid.ColumnSpan="2"
-                Height="20"
+                Height="24"
                 Background="{ThemeResource AccentFillColorDefaultBrush}"
```

Comment:
`ControlStrongStrokeColorDefault` is a bit softer, and the same color used by the `ProgressBar` which is very similiar

Reply:
Maybe using these will work?

Reply:
Yep! Those are ProgressBar specific brushes that reference AccentFillColorDefaultBrush and ControlStrongStrokeColorDefault. So yeah, doesn't matter which one you'd use.

Reply:
**ControlStrongStrokeColorDefault** is intended as a _border_. Note that the track of a ProgressBar is only a line of 1.5 px high. This color is way too dark, and not the color used in the things used in Windows (either in Explorer or Settings)
![image](https://github.com/microsoft/PowerToys/assets/65828559/31a4e0d1-7481-4072-a9c9-102a839cf601)
We need a _background_ color.

Reply:
Wait, did you mean the blue part or the gray part?

Reply:
The gray part.. but yeah, I agree the `ControlStrongStrokeColorDefault` is too dark looking at your screenshot. If there's a lighter gray in one of the brushes (to mimic what W11 Settings does) I think we should be good!